my-project/src/Entity/Product.php

```diff
@@ -27,12 +28,12 @@ class Product
     private $description;
 
     /**
-     * @ORM\Column(type="date")
+     * @ORM\Column(type="date", nullable=true)
      */
     private $add_date;
 
     /**
-     * @ORM\Column(type="date")
+     * @ORM\Column(type="date", nullable=true)
```

Comment:
Pole nie powinno być nullable, popraw proszę (i nie zapomnij o zrobieniu migracji, żeby to też się zmieniło w bazie)

Reply:
To samo dotyczy pola add_data